tools/stylelint/theme-mixin-api.ts

```diff
@@ -147,7 +147,12 @@ const plugin = (isEnabled: boolean, _options: never, context: {fix: boolean}) =>
       }
 
       const expectedProperty = type === 'density' ? '$density-scale' : '$config';
-      const expectedValue = `mat-get-${type}-config($config-or-theme)`;
+      const expectedValues = [`mat-get-${type}-config($config-or-theme)`];
```

Comment:
This is nitpicky so feel free to ignore: it should be slightly faster to use a `Set` here instead of an array since checking the set is constant time.

Reply:
Normally I'd agree, but in this case its a very small size and I do actually care about the order, so I'll leave it a list.